personalized_active_learning/datasets/types.py

```diff
@@ -57,7 +57,7 @@ def __getitem__(self, index: np.ndarray) -> Tuple[TextFeaturesBatch, torch.Tenso
             text_ids=torch.tensor(batch_text_ids),
             annotator_ids=torch.tensor(batch_annotator_ids),
             raw_texts=self.raw_texts[batch_text_ids],
-            embeddings=torch.tensor(self.embeddings[batch_text_ids]),
+            embeddings=self.embeddings[batch_text_ids],  # Is already a tensor
```

Comment:
Are we sure that its always a tensor?

Reply:
Yes sir. Especially after refactor, text embeddings - `TextFeaturesBatchDataset` will be only created by `BaseDataset` which will use `EmbeddingsCreator` to create tensor of embeddings.

P.S. Errors like that should be detected by typing however we still have many "blind points".